logparser/logger.py

```diff
@@ -0,0 +1,304 @@
+# Log Parser for RTI Connext.
+#
+#   Copyright 2016 Real-Time Innovations, Inc.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""Logger.
+
+The module contains the class to log the messages.
+"""
+
+
+class Logger(object):
+    """Class to log the messages.
+
+    Attributes:
+        COLORS: colors to use in the logs
+        KIND_TO_COLOR: logs messages to color
+        state (:obj:`list` of `str`): information about the parse process
+        verbosity (int): verbosity level of the log
+        inline (bool): show warnings/erros in network logs
+        ignorePackets (bool): ignore network events
+        ignore
+        showColors (bool): show colors in the log
+        formatDevice (:obj:`FormatDevice`): format device to print the logs
+        highlight (:obj:`compiled re`): show in bold regex matched logs
+        onlyIf (:obj:`compiled re`): show only
```

Comment:
What about this one which is similar to the *highlight* description?
*onlyIf (:obj:`compiled re`): show only regex matched logs*